### PR TITLE
Customizing for our needs

### PR DESCRIPTION
**Pull request by gogreen53**

### CODE REVIEW

**Comment by gogreen53** [comment]
@timhudson @tonyghita

File: lib/express-statsd.js [hunk 1]
```diff
@@ -18,16 +15,17 @@ module.exports = function expressStatsdInit (options) {
...
-      var key = req[options.requestKey];
-      key = key ? key + '.' : '';
+      var keys = generateStatNames(req.route);
 
-      // Status Code
-      var statusCode = res.statusCode || 'unknown_status';
-      client.increment(key + 'status_code.' + statusCode);
+      keys.forEach(function(key) {
+        // Status Code
+        var statusCode = res.statusCode || 'unknown_status';
```
Comment (dragonpaw) [inline lib/express-statsd.js]: I'd put this outside the loop since the value isn't dependent on the key.

File: lib/express-statsd.js [hunk 2]
```diff
@@ -18,16 +15,17 @@ module.exports = function expressStatsdInit (options) {
...
+        client.increment([key, 'status_code', statusCode].join('.'));
 
-      // Response Time
-      var duration = new Date().getTime() - startTime;
-      client.timing(key + 'response_time', duration);
+        // Response Time
+        var duration = new Date().getTime() - startTime;
```
Comment (dragonpaw) [inline lib/express-statsd.js]: Same.

**Comment by gogreen53** [comment]
:+1: from ash in slack

**Pull request merged**